### PR TITLE
Add Order Processing for Javascript:

### PR DESCRIPTION
**Pull request by XavierGeerinck**
Signed-off-by: Xavier Geerinck <xavier.geerinck@gmail.com>

### CODE REVIEW

File: pub_sub/javascript/checkout/package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "name": "checkout",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "type": "module",
+    "scripts": {
+        "start": "node index.js",
+        "start:dapr": "dapr run --app-id checkout --app-protocol http --dapr-grpc-port 3501 --components-path ./components -- npm run start",
```
Comment (greenie-msft) [inline pub_sub/javascript/checkout/package.json]: @XavierGeerinck should this be http?
```suggestion
        "start:dapr": "dapr run --app-id checkout --app-protocol http --dapr-http-port 3501 --components-path ./components -- npm run start",
```
```suggestion
        "start:dapr": "dapr run --app-id checkout --app-protocol http --dapr-grpc-port 3501 --components-path ./components -- npm run start",
```

File: pub_sub/javascript/checkout/index.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { DaprClient } from 'dapr-client';
+import { v4 as uuidv4 } from 'uuid';
+
+const SIDECAR_HOST = process.env.SIDECAR_HOST || "127.0.0.1";
```
Comment (amulyavarote) [inline pub_sub/javascript/checkout/index.js]: Can we change the variable name to DAPR_HOST

File: pub_sub/javascript/checkout/index.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+const SIDECAR_PORT = process.env.SIDECAR_PORT || 3500;
```
Comment (amulyavarote) [inline pub_sub/javascript/checkout/index.js]: Can we change the variable name to DAPR_PORT

File: pub_sub/javascript/order-processor/index.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+// Order Processing = Server which processes items
+// Checkout will do the pub sub
+import { DaprServer } from 'dapr-client';
+
+const SIDECAR_HOST = process.env.SIDECAR_HOST || "127.0.0.1";
```
Comment (amulyavarote) [inline pub_sub/javascript/order-processor/index.js]: Can we change the variable name to DAPR_HOST

File: pub_sub/javascript/order-processor/index.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+const SIDECAR_PORT = process.env.SIDECAR_PORT || 3500;
```
Comment (amulyavarote) [inline pub_sub/javascript/order-processor/index.js]: Can we change the variable name to DAPR_PORT?

File: pub_sub/javascript/checkout/index.js (hunk 2 above)
Comment (greenie-msft) [reply pub_sub/javascript/checkout/index.js]: Agreed, that would be consistent with the other examples

**Review by amulyavarote — APPROVED** [review]

**Pull request merged**